uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader.php

```diff
@@ -85,9 +86,72 @@ public function install() {
     $this->deleteUnneededCustomGroups();
     $this->createDefaultRelationshipTypes();
     $this->makeAllCurrenciesAvailable();
+    $this->enableComponentsAndFilterDefaultComponents();
     $this->runAllUpgraders();
   }
 
+  /**
+   * Enables required components and remove unwanted core components
+   * from the default components list.
+   */
+  public function enableComponentsAndFilterDefaultComponents() {
+    $this->setComponentsStatus();
+  }
+
+  /**
+   * Enables required components.
+   */
+  public function enableComponents() {
```

Comment:
Same thing here. If the method only enables the required components, the method name should say that, otherwise it means all